sinks/src/main/scala/com.snowplowanalytics.snowplow.eventgen/Config.scala

```diff
@@ -101,17 +126,12 @@ object Config {
     */
   def parse(argv: Seq[String]): Either[String, Cli] =
     application.parse(argv).leftMap(_.show).flatMap {
-      case RawCli(Some(path), output, region) =>
+      case Some(path) =>
         for {
           raw    <- loadFromFile(path)
           parsed <- parser.decode[Config](raw).leftMap(e => s"Could not parse config $path: ${e.show}")
-        } yield Cli(parsed, output, region)
-      case RawCli(None, output, region) =>
-        val raw = namespaced(ConfigFactory.load())
-        parser
-          .decode[Config](raw)
-          .leftMap(e => s"Could not resolve config without a provided hocon file: ${e.show}")
-          .map(Cli(_, output, region))
+        } yield Cli(parsed)
+      case _ => Left(s"Could not resolve config without a provided hocon file")
```

Comment:
Just a tip... you only need the `s"..."` syntax if you're doing parameter substitution in the string, i.e. `s"some $value"` or `"static text"`